_data-prepper/log-analytics.md

```diff
@@ -0,0 +1,149 @@
+* `aws_region` - If you use Amazon OpenSearch Service with AWS signing, set this value to your region.
+
+## FluentBit
+
+You will have to run FluentBit in your service environment. You can find the installation guide of FluentBit [here](https://docs.fluentbit.io/manual/installation/getting-started-with-fluent-bit). Ensure that you can configure the [FluentBit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to your Data Prepper Http Source. Below is an example `fluent-bit.conf` that tails a log file named `test.log` and forwards it to a locally running Data Prepper's http source, which runs by default on port 2021. 
+
+Note that you should adjust the file `path`, output `Host` and `Port` according to how and where you have FluentBit and Data Prepper running.
+
+### Example fluent-bit.conf without SSL and basic authentication enabled
+
+Example `fluent-bit.conf` without SSL and Basic Authentication enabled on the http source:
+
+```
+[INPUT]
+  name                  tail
+  refresh_interval      5
+  path                  test.log
+  read_from_head        true
+
+[OUTPUT]
+  Name http
+  Match *
+  Host localhost
+  Port 2021
+  URI /log/ingest
+  Format json
+```
+
+If your http source has SSL and basic authentication enabled, you will need to add the details of `http_User`, `http_Passwd`, `tls.crt_file`, and `tls.key_file` to the `fluent-bit.conf` as shown below.
+
+### Example fluent-bit.conf with SSL and basic authentication enabled
+
+Example `fluent-bit.conf` with SSL and basic authentication enabled on the http source:
+
+```
+[INPUT]
+  name                  tail
+  refresh_interval      5
+  path                  test.log
+  read_from_head        true
+
+[OUTPUT]
+  Name http
+  Match *
+  Host localhost
+  http_User myuser
+  http_Passwd mys3cret
+  tls On
+  tls.crt_file /full/path/to/certfile.crt
+  tls.key_file /full/path/to/keyfile.key
+  Port 2021
+  URI /log/ingest
+  Format json
+```
+
+# Next Steps
+
+Follow the [Log ingestion demo guide](https://github.com/opensearch-project/data-prepper/blob/main/examples/log-ingestion/README.md) to go through a specific example of Apache log ingestion from `FluentBit -> Data Prepper -> OpenSearch` running through Docker.
+
+In the future, Data Prepper will contain additional sources and processors which will make more complex log analytic pipelines available. Check out our [Roadmap](https://github.com/opensearch-project/data-prepper/projects/1) to see what is coming.  
+
+If there is a specifc source, processor, or sink that you would like to include in your log analytic workflow, and it is not currently on the Roadmap, please bring it to our attention by making a Github issue. Additionally, if you are interested in contributing, see our [Contribuing guidelines](../CONTRIBUTING.md) as well as our [Developer guide](developer_guide.md) and [Plugin development guide](plugin_development.md).
```

Comment:
```suggestion
If there is a specific source, processor, or sink that you would like to include in your log analytic workflow and is not currently on the roadmap, please bring it to our attention by creating a GitHub issue. Additionally, if you are interested in contributing to Data Prepper, see our [Contribuing Guidelines](../CONTRIBUTING.md) as well as our [Developer guide](developer_guide.md) and [Plugin development guide](plugin_development.md).
```

Reply:
Please correct the last two links and ensure they reflect the capitalization of the title of the destination.

Reply:
Added these changes manually.